flutter_local_notifications/example/lib/main.dart

```diff
@@ -183,12 +182,11 @@ class _HomePageState extends State<HomePage> {
   @override
   void initState() {
     super.initState();
-    _requestPermissions();
```

Comment:
Can this be kept so it request permissions so it's essentially done on startup? This was originally done as helps those using the example app to get started with testing the functionality to display notifications immediately